doc/cascadia/profiles.schema.json

```diff
@@ -0,0 +1,505 @@
+            "minLength": 1,
+            "type": "string"
+          },
+          "background": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the background color of the color scheme."
+          },
+          "black": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI black."
+          },
+          "blue": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI blue."
+          },
+          "brightBlack": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI bright black."
+          },
+          "brightBlue": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI bright blue."
+          },
+          "brightCyan": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI bright cyan."
+          },
+          "brightGreen": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI bright green."
+          },
+          "brightPurple": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI bright purple."
+          },
+          "brightRed": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI bright red."
+          },
+          "brightWhite": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI bright white."
+          },
+          "brightYellow": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI bright yellow."
+          },
+          "cyan": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI cyan."
+          },
+          "foreground": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the foreground color of the color scheme."
+          },
+          "green": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI green."
+          },
+          "purple": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI purple."
+          },
+          "red": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI red."
+          },
+          "white": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI white."
+          },
+          "yellow": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the color used as ANSI yellow."
+          }
+        },
+        "type": "object"
+      },
+      "type": "array"
+    }
+  },
+  "required": [
+    "profiles",
+    "schemes"
+  ],
+  "type": "object"
```

Comment:
we still have the problem where `defaultProfile` can be inside globals or outside of it, `keybindings` can be inside globals or outside, etc.
Anything that can exist in `globals` can also exist _outside_ of globals, so we need some sort of reusable object mapping that says so.